core/src/main/java/org/apache/iceberg/CachingCatalog.java

```diff
@@ -30,23 +35,98 @@
 import org.apache.iceberg.catalog.TableIdentifier;
 import org.apache.iceberg.exceptions.AlreadyExistsException;
 import org.apache.iceberg.relocated.com.google.common.collect.ImmutableList;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
 
+/**
+ * Class that wraps an Iceberg Catalog to cache table's.
+ *
```

Comment:
Javadoc should use `<p>` on lines between paragraphs so that it renders with separate paragraphs.

Reply:
Fixed. We have a number of places that don't do this, but I'm guessing we don't want to cause that large diff just for `<p>` (in a separate PR that is).